tests/appsec/integrations/flask_tests/test_flask_remoteconfig.py

```diff
@@ -20,7 +20,7 @@
 
 
 def _get_agent_client():
-    parsed = parse.urlparse(tracer._writer.agent_url)
+    parsed = parse.urlparse(tracer._span_aggregagtor.writer.agent_url)
```

Comment:
```suggestion
    parsed = parse.urlparse(tracer._span_aggregator.writer.agent_url)
```
Looks like maybe a typo in a `sed` command? There are a bunch of files with this typo.